### PR TITLE
Small improves for record_main page

### PR DESCRIPTION
**Pull request by breeswish**
- Hide statistics section. It will be moved to record_statistics page in the future.
- Adjust column width. Fix https://github.com/vijos/vj4/issues/247
- Adjust filter section.

Demo:

image

@twd2 @iceb0y PTAL

### CODE REVIEW

File: vj4/ui/templates/record_main.html [hunk 1]
```diff
@@ -92,6 +90,7 @@ <h1 class="section__title">{{ _('Filter') }}</h1>
       </div>
     </div>
     {% if statistics %}
+    <!--
```
Comment (twd2) [inline vj4/ui/templates/record_main.html]: ??? I like to see.

File: vj4/ui/pages/record_main.page.styl [hunk 2]
```diff
@@ -1,18 +1,21 @@
 .page--record_main
   .col--status
-    width: rem(150px)
+    width: rem(170px)
     position: relative
 
   .col--submit-at
-    width: rem(160px)
+    width: rem(100px)
```
Comment (twd2) [inline vj4/ui/pages/record_main.page.styl]: Bug: when a new record received via websocket, it will show full date time.

Comment (twd2) [inline vj4/ui/pages/record_main.page.styl]: Can decrease width of problem title.

Comment (breeswish) [inline vj4/ui/pages/record_main.page.styl]: Let's open another issue for new record's relative time.

Comment (twd2) [inline vj4/ui/pages/record_main.page.styl]: Did you test in English version?

Comment (breeswish) [inline vj4/ui/pages/record_main.page.styl]: Nop ^_^TAT

Comment (twd2) [inline vj4/ui/pages/record_main.page.styl]: Please add a demo in English version (English is longer ^\_^ T\_T).

**Pull request merged**